components/server/src/data/datamodel.json

```diff
@@ -5113,6 +5136,36 @@
                             "type": "integer"
                         }
                     ]
+                },
+                "completeness": {
+                    "name": "Metric type",
+                    "name_plural": "Metric types",
+                    "attributes": [
+                        {
+                            "name": "Subject type",
+                            "key": "subject_type"
+                        },
+                        {
+                            "name": "Metric type",
+                            "key": "metric_type"
+                        },
+                        {
+                            "name": "Missing from reports",
+                            "key": "reports",
+                            "url": "report_url"
+                        },
+                        {
+                            "name": "Status",
+                            "key": "status",
+                            "type": "status",
+                            "color": {
+                                "target_met": "positive",
+                                "near_target_met": "warning",
+                                "target_not_met": "negative",
+                                "technical_debt_target_met": "active"
+                            }
+                        }
```

Comment:
How can missing metrics have a status?

Reply:
The way it's implemented right now, it's the status of the entity, not the status of the metric. The status is always "target not met" since a metric that is missing is bad. Alternatively, this could indeed be left empty. What do you think?

Reply:
In this case, entities are metrics that are missing in a report. These missing metrics can't have a status *at all*, not even "target not met", since they don't exist. So I wouldn't use color for the entities.

Reply:
true... status removed